const.go

```diff
@@ -1,18 +1,14 @@
 package dstask
 
-import (
-	"os"
-	"os/user"
-	"path"
-	"strings"
-)
+import "os"
+
+func init() {
+	if os.Getenv("DSTASK_FAKE_PTY") == "1" {
```

Comment:
I think just checking this is set to anything is best, as it might be confusing if set to some thing else (eg, `yes` is common)